src/Overlay.vue

```diff
@@ -39,6 +39,12 @@
 <script>
 import inspectorOptions from "virtual:vue-inspector-options"
 
+const importMetaUrl = new URL(import.meta.url)
+const protocol = inspectorOptions.serverOptions?.https ? "https:" : importMetaUrl.protocol
+const host = inspectorOptions.serverOptions?.host ?? importMetaUrl.hostname
+const port = inspectorOptions.serverOptions?.port ?? importMetaUrl.port
+const baseUrl = `${protocol}//${host}:${port}`
```

Comment:
The following implementation was used as a reference.

https://github.com/vitejs/vite/blob/main/packages/vite/src/client/client.ts